addons/web/static/src/js/views/graph/graph_renderer.js

```diff
@@ -1,1099 +1,1046 @@
-odoo.define('web.GraphRenderer', function (require) {
```

Comment:
should we use the new convention for the modules' name?

Reply:
Done